File: TextosJapones/src/main/java/org/jisho/textosJapones/controller/novels/NovelsController.java
package org.jisho.textosJapones.controller.novels;

import javafx.fxml.FXML;
import javafx.fxml.Initializable;
import javafx.scene.layout.AnchorPane;
import javafx.scene.layout.StackPane;
import org.jisho.textosJapones.controller.mangas.*;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

import java.net.URL;
import java.util.ResourceBundle;

public class NovelsController implements Initializable {

	private static final Logger LOGGER = LoggerFactory.getLogger(NovelsController.class);

	@FXML
	private AnchorPane apRoot;

	@FXML
	private StackPane stackPane;

	@FXML
	protected AnchorPane apConteinerRoot;

	@FXML
	private NovelsProcessarController processarController;


	public AnchorPane getRoot() {
		return apConteinerRoot;
	}

	public StackPane getStackPane() {
		return stackPane;
	}

	public void initialize(URL arg0, ResourceBundle arg1) {
		processarController.setControllerPai(this);
	}

	public static URL getFxmlLocate() {
		return NovelsController.class.getResource("/view/novels/Novel.fxml");
	}

}

File: ProcessaTexto/src/main/kotlin/br/com/fenix/processatexto/controller/novels/NovelsController.kt
package br.com.fenix.processatexto.controller.novels

import javafx.fxml.FXML
import javafx.fxml.Initializable
import javafx.scene.layout.AnchorPane
import javafx.scene.layout.StackPane
import org.slf4j.Logger
import org.slf4j.LoggerFactory
import java.net.URL
import java.util.*


class NovelsController : Initializable {

    @FXML
    private lateinit var apRoot: AnchorPane

    @FXML
    lateinit var stackPane: StackPane

    @FXML
    private lateinit var apConteinerRoot: AnchorPane

    @FXML
    private lateinit var importarController: NovelsImportarController

    @FXML
    private lateinit var processarController: NovelsProcessarController

    val root: AnchorPane get() = apConteinerRoot

    override fun initialize(arg0: URL?, arg1: ResourceBundle?) {
        importarController.controllerPai = this
        processarController.controllerPai = this
    }

    companion object {
        private val LOGGER: Logger = LoggerFactory.getLogger(NovelsController::class.java)
        val fxmlLocate: URL get() = NovelsController::class.java.getResource("/view/novels/Novel.fxml")
    }
}